### PR TITLE
Non Namespaced endpoints

### PR DESCRIPTION
**Pull request by sreevatsanraman**
Current PR
- [x] System logs endpoint
- [x] Transaction state endpoint
http://builds.cask.co/browse/CDAP-DUT886

Previous PR
- [x] Monitor endpoints

### CODE REVIEW

File: cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/TransactionHttpHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ */
+public class TransactionHttpHandlerTest extends AppFabricTestBase {
+
+  /**
+   * Tests invalidating a transaction.
+   * @throws Exception
+   */
+  @Test
+  public void testInvalidateTx() throws Exception {
+    TransactionSystemClient txClient = AppFabricTestBase.getTxClient();
```
Comment (bdmogal) [inline cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/TransactionHttpHandlerTest.java]: Shouldn't need the ``AppFabricTestBase`` class name here, right?

**Comment by bdmogal** [comment]
Minor comment. :+1:.

**Comment by anwar6953** [comment]
Just want to point out that /ping and /version should also be moved to v3 (they exist within v2).
```
<base>/v2/ping (AppFabricHttpHandler)
<base>/v2/version (VersionHandler)
```

Note that the following endpoints also exist. Should they be v3?
```
<base>/ping (PingHandler)
<base>/config/cdap (ConfigHandler)
<base>/config/hbase (ConfigHandler)
```

File: cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/TransactionHttpHandlerTest.java (hunk 1 above)
Comment (sreevatsanraman) [inline cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/TransactionHttpHandlerTest.java]: Fixed.

**Comment by sreevatsanraman** [comment]
Thanks @anwar6953  - PR coming soon.

**Comment by sreevatsanraman** [comment]
Addressed comments. Merging.

**Pull request merged**